sdk/keyvault/azure-security-keyvault-certificates/src/main/java/com/azure/security/keyvault/certificates/models/KeyVaultCertificateWithPolicy.java

```diff
@@ -43,8 +43,8 @@ public KeyVaultCertificateWithPolicy setProperties(CertificateProperties propert
      * Get the certificate policy of the certificate
      * @return the cer content.
      */
-    public CertificatePolicy getCertificatePolicy() {
-        return this.certificatePolicy;
+    public CertificatePolicy getPolicy() {
```

Comment:
CertificatePolicy -> policy , covered from the feedback ?

Reply:
yup, [here](https://microsoft.sharepoint.com/teams/AzureDeveloperExperience/Shared%20Documents/Service%20-%20KeyVault/Azure%20SDK%20-%20Service%20-KeyVault/Release.one#Certificates%20consistency%20meeting%203&section-id=727d79ca-765a-4572-a0e9-bf7323fb398b&page-id=88d10b80-647a-4ca2-aa1c-dd6c53d71735&end)